examples/Cassie/run_simple_sim_with_cassie_sensor.cc

```diff
@@ -0,0 +1,237 @@
+#include <memory>
+
+#include <gflags/gflags.h>
+#include "drake/multibody/rigid_body_plant/rigid_body_plant.h"
+#include "drake/multibody/rigid_body_tree_construction.h"
+#include "drake/manipulation/util/sim_diagram_builder.h"
+#include "drake/systems/lcm/lcm_publisher_system.h"
+#include "drake/systems/lcm/lcm_subscriber_system.h"
+#include "drake/lcm/drake_lcm.h"
+#include "drake/multibody/joints/floating_base_types.h"
+#include "drake/multibody/parsers/urdf_parser.h"
+// #include "drake/multibody/rigid_body_plant/drake_visualizer.h"
+#include "drake/multibody/rigid_body_tree.h"
+#include "drake/systems/analysis/simulator.h"
+#include "drake/systems/framework/diagram.h"
+#include "drake/systems/framework/diagram_builder.h"
+#include "drake/systems/analysis/runge_kutta2_integrator.h"
+#include "drake/systems/analysis/runge_kutta3_integrator.h"
+
+#include "systems/robot_lcm_systems.h"
+#include "dairlib/lcmt_robot_output.hpp"
+#include "dairlib/lcmt_robot_input.hpp"
+#include "systems/primitives/subvector_pass_through.h"
+
+#include "examples/Cassie/cassie_utils.h"
+
+#include "drake/systems/sensors/accelerometer.h"
+#include "drake/systems/sensors/gyroscope.h"
+#include "systems/sensors/sim_cassie_sensor_aggregator.h"
+
+namespace dairlib {
+
+using dairlib::systems::SubvectorPassThrough;
+using drake::systems::lcm::LcmSubscriberSystem;
+using drake::systems::lcm::LcmPublisherSystem;
+
+// Simulation parameters.
+DEFINE_double(timestep, 1e-4, "The simulator time step (s)");
+DEFINE_double(youngs_modulus, 1e8,
+              "The contact model's Young's modulus (Pa)");
+DEFINE_double(us, 0.7, "The static coefficient of friction");
+DEFINE_double(ud, 0.7, "The dynamic coefficient of friction");
+DEFINE_double(v_tol, 0.01,
+              "The maximum slipping speed allowed during stiction (m/s)");
+DEFINE_double(dissipation, 2, "The contact model's dissipation (s/m)");
+DEFINE_double(contact_radius, 2e-4,
+              "The characteristic scale of contact patch (m)");
+DEFINE_string(simulation_type, "compliant",
+              "The type of simulation to use: "
+              "'compliant' or 'timestepping'");
+DEFINE_double(dt, 1e-3, "The step size to use for "
+              "'simulation_type=timestepping' (ignored for "
+              "'simulation_type=compliant'");
+
+// Cassie model paramter
+DEFINE_bool(floating_base, false, "Fixed or floating base model");
+
+int do_main(int argc, char* argv[]) {
+  gflags::ParseCommandLineFlags(&argc, &argv, true);
+
+  drake::lcm::DrakeLcm lcm;
+  std::unique_ptr<RigidBodyTree<double>> tree;
+  if (FLAGS_floating_base)
+    tree = makeCassieTreePointer("examples/Cassie/urdf/cassie_v2.urdf",
+                                 drake::multibody::joints::kQuaternion);
+  else
+    tree = makeCassieTreePointer();
+
+  std::shared_ptr<RigidBodyFrame<double>> imu_frame =
```

Comment:
Question--is the IMU located at the origin of the pelvis?

Reply:
I'm not sure and I couldn't find it on the wiki. Just sent an email to Agility Robotics.

Reply:
Confirmed with Agility Robotics. The position in MuJoCo is correct. Also, the numbers are also available in Simulink simulator. 

e.g. https://github.com/osudrl/cassie-mujoco-sim/blob/master/model/cassie.xml#L86